app/models/video.rb

```diff
@@ -7,4 +8,8 @@ def self.search_by_title(string)
     return [] if string.empty?
     where("lower(title) LIKE ?", "%#{string.downcase}%").order(created_at: :asc)
   end
+  
+  def average_rating
+    reviews.count == 0 ? 0 : (reviews.sum(:rating).to_f/reviews.count.to_f).round(1)

```

Comment:
Nice, but ActiveRecord has built in `.average` method - read docs [here](http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-average)